charts/laravel/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 1.16.0
 description: A Helm chart for Kubernetes
 name: laravel
 type: application
-version: 1.3.8
+version: 1.4.0
```

Comment:
@piksida I guess, I made a mistake when ask you to bump minor version, this is just a patch. Please use `1.3.9`